Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -113,12 +113,14 @@ protected function configure()
             ->setDescription('Fixes a directory or a file')
             ->setHelp(<<<EOF
 The <info>%command.name%</info> command tries to fix as much coding standards
-problems as possible on a given file or directory:
+problems as possible on a given file or files in a given directory and its subdirectories:
 
     <info>php %command.full_name% /path/to/dir</info>
     <info>php %command.full_name% /path/to/file</info>
 
-The <comment>--verbose</comment> option show applied fixers. When using ``txt`` format (default one) it will also displays progress notification.
+The <comment>--format</comment> option can be used to set the output format of the results; ``txt`` (default one), ``xml`` or ``json``.
+
+The <comment>--verbose</comment> option will show the applied fixers. When using ``txt`` format it will also displays progress notifications.
```

Comment:
when using **the** ``txt`` format

Or alternatively.

when using ``txt`` **as** format